packages/flutter_hooks/lib/src/framework.dart

```diff
@@ -200,6 +200,7 @@ abstract class HookState<R, T extends Hook<R>> with Diagnosticable {
   @protected
   BuildContext get context => _element!;
   HookElement? _element;
+  VoidCallback? _onDidBuild;
```

Comment:
the private property should be next to the getter/setter for readability

Reply:
Done